Dockerfile

```diff
@@ -0,0 +1,106 @@
+# Build:
+#   docker build -t discourse/ruby:3.3-bookworm-slim --build-arg DEBIAN_RELEASE=bookworm --build-arg RUBY_VERSION=3.3.3 .
+#
+# You are not expected to build and push this Docker image by hand. This is done by the `build` Github Actions workflow.
+ARG DEBIAN_RELEASE=
+ARG RUBY_VERSION=
+
+FROM debian:${DEBIAN_RELEASE}-slim
+
+ARG RUBY_VERSION
+
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US.UTF-8
+ENV LC_ALL en_US.UTF-8
+ENV RUBY_VERSION=${RUBY_VERSION}
+
+# Installs system dependencies required to run Ruby
+RUN set -eux; \
+  DEBIAN_FRONTEND=noninteractive apt-get update; \
+  DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+  bzip2 \
```

Comment:
Nitpick: I would indent the name of the packages (i.e. the arguments of `apt-get`) just to make it clear that these commands are not invocated.

Reply:
My editor formats this automatically. Need to figure out how to turn this formatting off.